plugins/org.jkiss.dbeaver.ext.dameng/src/org/jkiss/dbeaver/ext/dameng/model/DamengUser.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.dameng.model;
+
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.dameng.DamengConstants;
+import org.jkiss.dbeaver.model.DBPDataSource;
+import org.jkiss.dbeaver.model.DBPObjectWithLongId;
+import org.jkiss.dbeaver.model.DBPScriptObject;
+import org.jkiss.dbeaver.model.access.DBAUser;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.meta.Property;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.struct.DBSObject;
+
+import java.sql.Timestamp;
+import java.util.Map;
+
+/**
+ * @author Shengkai Bai
+ */
+public class DamengUser implements DBAUser, DBPScriptObject, DBPObjectWithLongId {
+
+    private DamengDataSource dataSource;
+
+    private long id;
+
+    private String name;
+
+    private Timestamp createTime;
+    private Type type;
+
+    private LockedStatus lockedStatus;
+
+    private long tablespaceId;
+
+    public DamengUser(DamengDataSource dataSource, JDBCResultSet dbResult) {
+        this.dataSource = dataSource;
+        this.id = JDBCUtils.safeGetLong(dbResult, "ID");
```

Comment:
I think you can create some constants for the ID and NAME values. In the DamengConstants class.